recipes/sqlite.rb

```diff
@@ -0,0 +1,21 @@
+#
+# Author:: Ronald Doorn (<rdoorn@schubergphilie.com>)
+#
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+chef_gem 'sqlite3' do
+  compile_time true
+end
```

Comment:
You might want to take a look at this new method I added to the aws cookbook to dynamically install the gem when the provider attempts to and fails to do the require for the gem.

https://github.com/chef-cookbooks/aws/pull/231